src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.Project.ps1

```diff
@@ -4207,4 +4207,38 @@ class Project: ADOSVTBase
         }
         return $controlResult;
     }
+
+    hidden [ControlResult] CheckDisabledCreationOfClassicPipeline([ControlResult] $controlResult)
+    {
+        $controlResult.VerificationResult = [VerificationResult]::Failed
+        if($this.PipelineSettingsObj)
+        {
+            $orgLevelScope = $this.PipelineSettingsObj.disableClassicPipelineCreation.orgEnabled;
```

Comment:
Is $orgLevelScope check required? If the setting is enabled at org level, it is automatically enabled at project level and hence you will get true in $projLevelScope itself. Second check may be unnecessary. Please validate if the orgLevelScope use case flow will ever happen

Reply:
Yes, If the setting is enabled at org level, it is automatically enabled at project level. I referred the code of a similar control. For that control it is checked for both, org level and project level. So I too checked for both.

Reply:
Which control? Let me check if there is an API issue with Juhi

Reply:
You can keep only project. Remove code for checking org level

Reply:
okay